modules/compliance-increasing-operator-limits.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-increasing-operator-limits"]
+= Increasing Compliance Operator resource limits
+
+In some cases, the Compliance Operator might require more memory than the default limits allow. The best way to mitigate this issue is to set custom resource limits.
+
+To increase the default memory and CPU limits of scanner pods, see _`ScanSetting` Custom resource_.
+
+.Procedure
+
+. To increase the Operator's memory limits to 500Mi, create the following patch file:
++
+[source,yaml]
+----
+spec:
+  config:
+    resources:
+      limits:
+        memory: 500Mi
+----
+
+. Apply the patch file:
++
+[source,terminal]
+----
+$ oc patch sub compliance-operator -nopenshift-compliance --patch-file co-memlimit-patch.yaml --type=merge
```

Comment:
You don't specify the file name in the previous step, so it would be clearer to use a variable for the file name here. Consider making that change at some point.